make/_shared/tools/00_mod.mk

```diff
@@ -161,7 +161,7 @@ ADDITIONAL_TOOLS ?=
 tools += $(ADDITIONAL_TOOLS)
 
 # https://go.dev/dl/
-VENDORED_GO_VERSION := 1.22.5
+VENDORED_GO_VERSION := 1.22.8
```

Comment:
NOTE: this is an unrelated change

Reply:
Could we move this to a seperate PR? Such that the Release notes make sense.

Reply:
Yeah not a bad shout, done: https://github.com/cert-manager/cert-manager/pull/7406